samples/snippets/src/main/java/pubsublite/spark/AdminUtils.java

```diff
@@ -0,0 +1,198 @@
+                Topic.RetentionConfig.newBuilder()
+                    // How long messages are retained.
+                    .setPeriod(Durations.fromDays(1))
+                    // Set storage per partition to 30 GiB. This must be 30 GiB-10 TiB.
+                    // If the number of bytes stored in any of the topic's partitions grows
+                    // beyond this value, older messages will be dropped to make room for
+                    // newer ones, regardless of the value of `period`.
+                    .setPerPartitionBytes(30 * 1024 * 1024 * 1024L))
+            .setName(topicPath.toString())
+            .build();
+
+    AdminClientSettings adminClientSettings =
+        AdminClientSettings.newBuilder().setRegion(CloudRegion.of(cloudRegion)).build();
+
+    try (AdminClient adminClient = AdminClient.create(adminClientSettings)) {
+      Topic response = adminClient.createTopic(topic).get();
+      System.out.println(response.getAllFields() + "created successfully.");
+    }
+  }
+
+  public static void createSubscriptionExample(
+      String cloudRegion, char zoneId, String projectId, String topicId, String subscriptionId)
+      throws Exception {
+
+    TopicPath topicPath =
+        TopicPath.newBuilder()
+            .setProject(ProjectId.of(projectId))
+            .setLocation(CloudZone.of(CloudRegion.of(cloudRegion), zoneId))
+            .setName(TopicName.of(topicId))
+            .build();
+
+    SubscriptionPath subscriptionPath =
+        SubscriptionPath.newBuilder()
+            .setLocation(CloudZone.of(CloudRegion.of(cloudRegion), zoneId))
+            .setProject(ProjectId.of(projectId))
+            .setName(SubscriptionName.of(subscriptionId))
+            .build();
+
+    Subscription subscription =
+        Subscription.newBuilder()
+            .setDeliveryConfig(
+                // Possible values for DeliveryRequirement:
+                // - `DELIVER_IMMEDIATELY`
+                // - `DELIVER_AFTER_STORED`
+                // You may choose whether to wait for a published message to be successfully written
+                // to storage before the server delivers it to subscribers. `DELIVER_IMMEDIATELY` is
+                // suitable for applications that need higher throughput.
+                Subscription.DeliveryConfig.newBuilder()
+                    .setDeliveryRequirement(
+                        Subscription.DeliveryConfig.DeliveryRequirement.DELIVER_IMMEDIATELY))
+            .setName(subscriptionPath.toString())
+            .setTopic(topicPath.toString())
+            .build();
+
+    AdminClientSettings adminClientSettings =
+        AdminClientSettings.newBuilder().setRegion(CloudRegion.of(cloudRegion)).build();
+
+    try (AdminClient adminClient = AdminClient.create(adminClientSettings)) {
+      Subscription response = adminClient.createSubscription(subscription).get();
+      System.out.println(response.getAllFields() + "created successfully.");
+    }
```

Comment:
Ditto. Can you catch and ignore `com.google.api.gax.rpc.AlreadyExistsException` errors here.